### PR TITLE
add more units - match what wildfly and rhq support

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

File: api/src/main/java/org/hawkular/inventory/api/model/MetricUnit.java [hunk 1]
```diff
@@ -26,13 +26,54 @@
...
     SECONDS("s"),
-    MINUTE("min"),
-    BYTE("b"),
-    KILO_BYTE("kb");
+    MINUTES("m"),
+    HOURS("h"),
+    DAYS("d"),
+
+    // Rate
+    PER_JIFFY("/j"),
```
Comment (metlos) [inline api/src/main/java/org/hawkular/inventory/api/model/MetricUnit.java:64]: Are these really useful? If we are going to support derived units, we should maybe support them a little bit more elaborately than this.

Comment (pilhuhn) [inline api/src/main/java/org/hawkular/inventory/api/model/MetricUnit.java:64]: Jiffy is a strange thingy anyway :)

There is JSR 363 that may be helpful here.

**Comment by pilhuhn** [comment]
I wonder a) how much to align things here with JSR 363 and b) how much of this can/has to be shared with -metrics, as at one point in time they also want to tag metrics with units and the logic to up- and down-scale them.

**Comment by jmazzitelli** [comment]
Note that these come straight from WildFly MeasurementUnit. If we want to support a subset of them, that's OK. I just figured we want to at least match the names of what is there (even if its a subset).

It will make mapping between them easy:

MeasurementUnit.valueOf(inventoryMetricUnit.name)

and vice versa.

**Comment by metlos** [comment]
I'm a little bit concerned about using JSR 363 right now, because the impl  is not GA and the standard has not been approved yet.

The implementation is here: https://github.com/unitsofmeasurement/uom-se
The API is here: https://github.com/unitsofmeasurement/unit-api
and Jackson (de)serializer is here: https://github.com/unitsofmeasurement/jackson-module-unitsofmeasure

All of them seem to be at various levels of completion.

I think I'm inclined to just include Wfly's units as @jmazzitelli proposes, mainly out of the fear of unknown ;)

That being said, if we decide to go down that route, could you please make this pass the CI, Mazz?

**Comment by jmazzitelli** [comment]
> That being said, if we decide to go down that route, could you please make this pass the CI, Mazz?

I just saw this now. This is no longer easily merged because of renamed/moved directories. I will create another PR with this same diff but based on master and close this one.

**Comment by jmazzitelli** [comment]
see PR#145 as the replacement to this one

**Pull request closed without merging**